client/src/pages/SignupPage/SignupPage.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+
+import { Formik, Field, Form, ErrorMessage } from 'formik';
+import request from "../../utils/request";
+import HeaderComponent from "../../components/HeaderComponent/HeaderComponent";
+import FooterComponent from "../../components/FooterComponent/FooterComponent";
+import "./SignupPage.css";
+
+const SignupPage = () => {
+  const initialValues = {
+    name: "",
+    email: "",
+    phone: "",
+    address: "",
+    password: "",
+    confirmPassword: "",
+  };
+
+  const handleSubmit = async (values, { setSubmitting, resetForm, setStatus }) => {
+    try {
+      //Call API đăng ký
+      const response = await request('../../../../server/routes/auth', 'post', values);
+      setStatus({ success: true, message: response.message });
+      resetForm();
+      //Chuyển sang trang đăng nhập khi đăng ký thành công
+      window.location.href = "/login";
```

Comment:
Chỗ này em nên sử dụng như sau:
```suggestion
      const navigate = useNavigate();
      
      navigate('/login', { replace: true })
```

Reply:
đã sửa